packages/forma-36-website/src/components/Container.js

```diff
@@ -49,34 +49,34 @@ const styles = {
 };
 
 const markToComponentMap = {
-  h1: props => <DisplayText {...props} />,
-  h2: props => <Heading element="h2" {...props} />,
-  h3: props => <Subheading element="h3" {...props} />,
-  h4: props => <Subheading element="h4" {...props} />,
-  h5: props => <Subheading element="h5" {...props} />,
-  h6: props => <Subheading element="h6" {...props} />,
-  p: props => <Paragraph className="f36-font-size--l" {...props} />,
-  a: props => <TextLink {...props} />,
-  ul: props => <List className="f36-margin-bottom--m" {...props} />,
-  li: props => (
+  h1: (props) => <DisplayText {...props} />,
+  h2: (props) => <Heading element="h2" {...props} />,
+  h3: (props) => <Subheading element="h3" {...props} />,
+  h4: (props) => <Subheading element="h4" {...props} />,
+  h5: (props) => <Subheading element="h5" {...props} />,
+  h6: (props) => <Subheading element="h6" {...props} />,
+  p: (props) => <Paragraph className="f36-font-size--l" {...props} />,
+  a: (props) => <TextLink className="f36-font-size--l" {...props} />,
```

Comment:
Poop, this doesn't work, font size remains unchanged. Seems to need `!important` in order to trump the styles

Reply:
damn, I will use `style` to force it then

Reply:
@m10l checked, now it is working.